mmv1/products/cloudfunctions2/api.yaml

```diff
@@ -0,0 +1,302 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: CloudFunctions2
+display_name: Cloud Functions (2nd gen)
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://cloudfunctions.googleapis.com/v2/
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Cloud Functions API
+    url: https://console.cloud.google.com/apis/library/cloudfunctions.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: 'CloudFunction'
```

Comment:
Just confirming- you want `google_cloudfunctions2_cloud_function` over `google_cloudfunctions2_function` or `google_cloudfunctions2_cloudfunction`, right? We can amend this value if either of those would be preferable.